src/features/projects/screens/SingleProjectDetails.tsx

```diff
@@ -11,7 +11,7 @@ import { useLocale, useTranslations } from 'next-intl';
 import CancelIcon from '../../../../public/assets/images/icons/CancelIcon';
 import ExpandIcon from '../../../../public/assets/images/icons/ExpandIcon';
 import ProjectInfo from '../components/projectDetails/ProjectInfo';
-import ProjectSnippet from '../../projectsV2/components/ProjectSnippet';
+import ProjectSnippet from '../../projectsV2/ProjectSnippet';
```

Comment:
Why are we using the ProjectSnippet from v2 here, instead of the original project snippet?

Reply:
replaced it with the old project snippet